content/pages/constitution.md

```diff
@@ -0,0 +1,313 @@
+Author: Fatema Boxwala
+Title: Constitution
+hidden: true
+
+# Structure of Organization #
+
+The Women in Computer Science Undergraduate Subcomittee is a
+subcommittee of Women in Computer Science, a standing committee of the
+Cheriton School of Computer Science. For the remainder of this document,
+this standing committee will be referred to as the "WiCS Committee". All
+references to: "the undergraduate committee", "the committee", and
+"committee members" refer to the Undergraduate Subcommittee unless
+otherwise specified.
+
+## Purpose and Vision ##
+
+The undergraduate committee aims to create a safe environment for cis
+and trans women as well as non-binary individuals in computer science,
+and to advocate for the inclusion of these marginalized groups in the
+computer science community at large.
+
+## Representation ##
+
+The Women in Computer Science Undergraduate Subcommittee aims to
+represent and support undergraduate cis and trans women, and non-binary
+individuals in computer science. Additionally, we seek to include and
```

Comment:
"in computer science" -> "enrolled in computer science and related undergraduate programs at the University of Waterloo"